Revise preview section in README.md (#2430)
elastic/security-docs: README.md

### Preview documentation changes

> When you open a pull request, preview links are automatically added as a comment on the PR. Once the CI build is complete, the links will be live and you can follow them to preview your changes.

Reviewer: ```suggestion
When you open a pull request, preview links are automatically added as a comment in the PR. Once the CI check builds successfully, the links will be live and you can click them to preview your changes.
```
